plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleDependencyResolutionHelper.java

```diff
@@ -90,7 +92,8 @@ public static String determinePluginVersion() {
      * @return true if the descriptor represents a Gradle project, false otherwise.
      */
     public static boolean isProject(Project project, DependencyDescriptor descriptor) {
-        final String specGAV = String.format("%s:%s:%s", descriptor.getGroup(), descriptor.getName(), descriptor.getVersion());
+        final String specGAV = String.format(GROUP_ARTIFACT_VERSION_FORMAT, descriptor.getGroup(), descriptor.getName(),
```

Comment:
The string was duplicated multiple times in the file, hence I had to externalize it as a reference.